ParkJInjoo/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "parkjinjoo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^1.0.3",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql2": "^2.3.3",
+    "nodemon": "^2.0.20",
+    "typeorm": "^0.3.10"
+  }
```

- 프로젝트에서 3rd-party 패키지를 설치해서 사용할 때에는 주의해야 될 사항이 있습니다.
- 해당 패키지가 production 레벨(사용자에게 배포)에서 사용될 패키지인지 local 개발용도로만 사용되는 패키지인지를 알고 있어야 합니다.
- express, mysql, typeorm, dotenv, cors와 같은 패키지는 사용자에게 서비스되는 코드에서는 반드시 포함되어야 하는 라이브러리이므로 package.json 내에 `dependencies`에 포함되어야 합니다.
- 하지만, nodemon과 같은 패키지는 로컬에서 개발할 때, 생산성을 위해서 사용하는 패키지입니다. 그렇기 때문에 이는 `devDependencies`에 추가 혹은 `global 환경`에 설치해서 사용할 수 있도록 만들어주세요.
```suggestion
  "dependencies": {
    "cors": "^2.8.5",
    "dotenv": "^16.0.3",
    "express": "^4.18.2",
    "mysql2": "^2.3.3",
    "typeorm": "^0.3.10"
  },
  devDependencies: {
      "morgan": "^1.10.0",
      "nodemon": "^2.0.20"
  }
```